docs/federation.md

```diff
@@ -75,7 +75,7 @@ app.listen(3000)
 
 ### Federation with \_\_resolveReference caching
 
-Just like standard resolvers, the `__resolveReference` resolver can be a performance bottleneck. To avoid this, the it is strongly recommended to define the `__resolveReference` function for an entity as a [loader](#defineLoaders).
+Just like standard resolvers, the `__resolveReference` resolver can be a performance bottleneck. To avoid this, the it is strongly recommended to define the `__resolveReference` function for an entity as a [loader](../#loaders).
```

Comment:
```suggestion
Just like standard resolvers, the `__resolveReference` resolver can be a performance bottleneck. To avoid this, the it is strongly recommended to define the `__resolveReference` function for an entity as a [loader](docs/loaders).
```

Reply:
Thanks I wasn't sure about this link !